python/cuml/cuml/tests/test_sklearn_import_export.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import numpy as np
+
+from cuml.cluster import KMeans, DBSCAN
+from cuml.decomposition import PCA, TruncatedSVD
+from cuml.linear_model import (
+    LinearRegression,
+    LogisticRegression,
+    ElasticNet,
+    Ridge,
+    Lasso,
+)
+from cuml.manifold import TSNE
+from cuml.neighbors import NearestNeighbors
+
+from cuml.testing.utils import array_equal
+
+from numpy.testing import assert_allclose
+
+from sklearn.datasets import make_blobs, make_classification, make_regression
+from sklearn.utils.validation import check_is_fitted
+from sklearn.cluster import KMeans as SkKMeans, DBSCAN as SkDBSCAN
+from sklearn.decomposition import PCA as SkPCA, TruncatedSVD as SkTruncatedSVD
+from sklearn.linear_model import (
+    LinearRegression as SkLinearRegression,
+    LogisticRegression as SkLogisticRegression,
+    ElasticNet as SkElasticNet,
+    Ridge as SkRidge,
+    Lasso as SkLasso,
+)
+from sklearn.manifold import TSNE as SkTSNE
+from sklearn.neighbors import NearestNeighbors as SkNearestNeighbors
+
+###############################################################################
+#                              Helper functions                               #
+###############################################################################
+
+
+@pytest.fixture
+def random_state():
+    return 42
```

Comment:
Do we really need this instead of using `42` in the tests directly?

We could have a global version of this that allows us to run the tests with several seeds, but maybe something to tackle in the future/new PR

Reply:
We don't really need it at all